store/migrations/README.md

```diff
@@ -1,86 +0,0 @@
-## Migrations
```

Comment:
Happy to get rid of this. Looks like the actual cldev migration command is still lying around to. Let's axe it.